akka-http2-support/src/main/scala/akka/http/impl/engine/http2/FrameRenderer.scala

```diff
@@ -18,6 +18,20 @@ object FrameRenderer {
 
   def render(frame: FrameEvent): ByteString =
     frame match {
+      case GoAwayFrame(lastStreamId, errorCode, debug) ⇒
+        val bb = new ByteStringBuilder
+        // TODO: render this more correct at the moment it will just kill the reserved bit
+        bb.putInt(0x0FFFFFFF & lastStreamId)
```

Comment:
This will kill more than the reserved bit. Maybe just add a requirement in `GoAwayFrame` to prevent negative stream ids  in the first place?